apps/web/assets/js/components/Opportunity.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+const translated_difficulty = (level) => {
+  var term;
```

Comment:
This is a nitpick, but IMO you're better off just returning directly from the `case` branches of the `switch` rather than declaring the variable at the top, mutating it, and returning.

Something like this:

```js
const translatedDifficulty = level => {
  switch (level) {
    case 1: 
      return "Starter";
    case 5:
      return "Intermediate";
    case 9:
      return "Advanced";
    default:
      return "probably want some base case here just so things don't break"
  }
}
```

It has the added benefit of being more like how you'd do it in Elixir anyway.

Reply:
Sounds good @cafreeman!  Before Elixir, I did Ruby and Java where multiple returns were frowned upon 😁